sympy/galgebra/printing.py

```diff
@@ -30,10 +30,14 @@
 
 import os
 import sys
+
```

Comment:
There was an empty line above ```from sympy import```, and I thought "well if we're grouping imports here, then the grouping should show some form of consistency".